Building the docs (and thus the debs) in ubuntu precise fails if python-mako is installed

If python-mako is installed on ubuntu precise is installed, then building the debian packages, specifically the docs, will fail with the following... (this is as of b3a029dd8a836f57a4e9d53c305fb42dc29c6647)


<pre>
jtang@x01:~/develop/salt$ fakeroot debian/rules binary
make -C doc html
make[1]: Entering directory `/home/jtang/develop/salt/doc'
No need to update translations. Skipping...
sphinx-build -b html -d _build/doctrees    . _build/html
Making output directory...
Running Sphinx v1.1.3
/usr/lib/python2.7/dist-packages/pygments/plugin.py:39: UserWarning: Module M2Crypto was already imported from /dev/null, but /usr/lib/python2.7/dist-packages is being added to sys.path
  import pkg_resources
/usr/lib/python2.7/dist-packages/pygments/plugin.py:39: UserWarning: Module mako was already imported from /dev/null, but /usr/lib/python2.7/dist-packages is being added to sys.path
  import pkg_resources
loading translations [en]... locale not available
loading pickled environment... not yet created
[autosummary] generating autosummary for: contents.rst, faq.rst, index.rst, ref/auth/all/index.rst, ref/auth/all/salt.auth.auto.rst, ref/auth/all/salt.auth.keystone.rst, ref/auth/all/salt.auth.ldap.rst, ref/auth/all/salt.auth.pam.rst, ref/auth/all/salt.auth.stormpath_mod.rst, ref/cli/index.rst, ..., topics/tutorials/states_pt1.rst, topics/tutorials/states_pt2.rst, topics/tutorials/states_pt3.rst, topics/tutorials/states_pt4.rst, topics/tutorials/walkthrough.rst, topics/tutorials/walkthrough_macosx.rst, topics/tutorials/writing_tests.rst, topics/virt/disk.rst, topics/virt/index.rst, topics/virt/nic.rst
loading intersphinx inventory from http://docs.python.org/3/objects.inv...
loading intersphinx inventory from http://docs.python.org/2/objects.inv...
building [html]: targets for 623 source files that are out of date
updating environment: 623 added, 0 changed, 0 removed
reading sources... [100%] topics/virt/nic
looking for now-outdated files... none found
pickling environment... done
checking consistency... done
preparing documents... done
writing output... [ 98%] topics/tutorials/states_ordering
Exception occurred:
  File "/usr/lib/python2.7/dist-packages/docutils/nodes.py", line 173, in walkabout
    if child.walkabout(visitor):
  File "/usr/lib/python2.7/dist-packages/docutils/nodes.py", line 173, in walkabout
    if child.walkabout(visitor):
  File "/usr/lib/python2.7/dist-packages/docutils/nodes.py", line 173, in walkabout
    if child.walkabout(visitor):
  File "/usr/lib/python2.7/dist-packages/docutils/nodes.py", line 165, in walkabout
    visitor.dispatch_visit(self)
  File "/usr/lib/python2.7/dist-packages/docutils/nodes.py", line 1604, in dispatch_visit
    return method(node)
  File "/usr/lib/python2.7/dist-packages/sphinx/writers/html.py", line 249, in visit_literal_block
    **highlight_args)
  File "/usr/lib/python2.7/dist-packages/sphinx/highlighting.py", line 188, in highlight_block
    lexer = lexers[lang] = get_lexer_by_name(lang)
  File "/usr/lib/python2.7/dist-packages/pygments/lexers/__init__.py", line 77, in get_lexer_by_name
    for cls in find_plugin_lexers():
  File "/usr/lib/python2.7/dist-packages/pygments/plugin.py", line 53, in find_plugin_lexers
    yield entrypoint.load()
  File "/usr/lib/python2.7/dist-packages/pkg_resources.py", line 1989, in load
    entry = __import__(self.module_name, globals(),globals(), ['__name__'])
ImportError: No module named ext.pygmentplugin
The full traceback has been saved in /tmp/sphinx-err-1gn6uG.log, if you want to report the issue to the developers.
Please also report this if it was a user error, so that a better error message can be provided next time.
Either send bugs to the mailing list at <http://groups.google.com/group/sphinx-dev/>,
or report them in the tracker at <http://bitbucket.org/birkenfeld/sphinx/issues/>. Thanks!
make[1]: *** [html] Error 1
make[1]: Leaving directory `/home/jtang/develop/salt/doc'
make: *** [binary] Error 2
</pre>

Uninstalling the python-mako package allows the docs to be built again.

--- a/doc/conf.py
+++ b/doc/conf.py
@@ -91,8 +91,6 @@ MOCK_MODULES = [
     # modules, renderers, states, returners, et al
     'django',
     'libvirt',
-    'mako',
-    'mako.template',
     'MySQLdb',
     'MySQLdb.cursors',
     'psutil',

